frontend/src/pages/Profile/index.js

```diff
@@ -1,108 +1,250 @@
-import Layout from '../../layout';
+import Fandom from '../../components/Profile/Fandom';
+import tempUserImage from '../../assets/images/temp_profile.svg';
+import panzee from '../../assets/images/fandom/panzee.svg';
+import virus from '../../assets/images/fandom/virus.svg';
+import fox from '../../assets/images/fandom/fox.svg';
+import elk from '../../assets/images/fandom/elk.svg';
+import dog from '../../assets/images/fandom/dog.svg';
+import bat from '../../assets/images/fandom/bat.svg';
+import pigeon from '../../assets/images/fandom/pigeon.svg';
+import developer from '../../assets/images/fandom/developer.svg';
+import achieve1 from '../../assets/images/achieve/achieve1.svg';
+import achieve2 from '../../assets/images/achieve/achieve2.svg';
+import achieve3 from '../../assets/images/achieve/achieve3.svg';
+import achieve4 from '../../assets/images/achieve/achieve4.svg';
+import achieve5 from '../../assets/images/achieve/achieve5.svg';
+import achieve6 from '../../assets/images/achieve/achieve6.svg';
+import achieve7 from '../../assets/images/achieve/achieve7.svg';
+
+import PageLayout from 'components/common/PageLayout';
 import {
   Body,
-  Button,
-  ButtonWrap,
+  BoxBody,
+  Challenge,
   Contents,
-  ContentsWrapper,
-  Info,
-  InfoCard,
-  InfoCardContents,
-  InfoCardTitle,
-  InfoFandom,
+  Graph,
   Left,
-  Name,
-  ProfileBannerImg,
-  ProfileImg,
-  ProfileInfo,
-  ProfileWrapper,
+  MFGList,
+  MFGTitle,
+  MyFavoriteGenre,
+  MyGameInfo,
+  Prefix,
+  RecentGame,
+  RecentGameImage,
+  RecentGameItem,
+  RecentGameList,
+  RecentGameTitle,
   Right,
-  SocialImage,
-  UserInfoWrapper,
+  UserImage,
+  UserInfo,
+  UserLevel,
   UserName,
-  WindowTitle,
+  UserWrap,
 } from './styles';
-import profileWrapperImage from '../../assets/images/login_main.png';
-import tempImage from '../../assets/images/naver_logo.png';
-import ProfileChallenge from '../../components/Profile/ProfileChallenge';
-import Fandom from '../../components/Profile/Fandom';
+import TitleBox from 'components/common/TitleBox';
+import { Doughnut } from 'react-chartjs-2';
+import { Chart, ArcElement, Tooltip } from 'chart.js';
+import { useState } from 'react';
+import { useEffect } from 'react';
```

Comment:
```suggestion
import { useState, useEffect } from 'react';
```
이렇게 한줄에 넣을 수 있고
다른 import는 비슷한 애들끼리 순서대로 나열하면 찾기 편할 것 같아요